### DISCUSSION TITLE
Challenge quest suggestions

### DISCUSSION THREAD

**Discussion by Severnate** [NillerMedDild/Enigmatica6#4550] [Ideas]
1. fully automated multi-effect high level potion production with wixies
2. full-auto general mob farm
3. Create/Ars Wood farm feeding a stirling dynamo for first power gen
4. a l l  t h e  b e e s
5. Tetra/Tinkers Tool that does [high amount]+ Damage per hit 
6. Automatic Sorting system that puts your items where they belong categorised however you fancy
7. Request based automation of all basic ingots/plates/rods

**Comment by NielsPilgaard** [comment]
Good suggestions, but most of these things are impossible to verify through quests :/

**Comment by theboo** [comment]
well we have precedence for this, most of our challenge type quests are just checkmark quests with no reward based on the honor system... like this one 
![image](https://user-images.githubusercontent.com/3585761/161366705-3143da9e-4bf5-43cf-8846-e5cbb1875c5a.png)

**Reply by NielsPilgaard** [reply]
Oh.. right 😅
